### DISCUSSION TITLE
Sweden is four price areas

### DISCUSSION THREAD

**Discussion by Blixtmannen** [electricitymaps/electricitymaps-contrib#3685] [Ideas] [answered]
I suggest Sweden is partitioned into the four electricity market price areas. Prices and production are ofter very different in the areas.

**Comment by VIKTORVAV99** [comment, marked as answer]
This has been a issue for a while now and there is no easy solution as the source for the data don't split it per bidding zone.

If Sweden where to be split it would turn gray on the map and I think we all agree that aggregated data is better than no data.

**Comment by VIKTORVAV99** [comment]
Now everything is finally ready for #3936 to be merged (pending review) which will split Sweden into the bidding zones.

That should add the proper price information for each zones and give a better representation of the actual co2eq intensity in Sweden.
